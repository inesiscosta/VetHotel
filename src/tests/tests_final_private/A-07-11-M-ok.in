8
5
h1
B1
b1
2
3
DECIDUOUS
5
h1
X1
x1
3
4
EVERGREEN
5
h1
A1
a1
6
7
EVERGREEN
5
h1
R1
r1
10
12
DECIDUOUS
6
h1
1
0
0
