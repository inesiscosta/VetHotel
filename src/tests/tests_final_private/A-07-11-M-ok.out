Main Menu
1 - New File
2 - Open File
3 - Save File
4 - Next Season
5 - Calculate Global Satisfaction
6 - Manage Animals
7 - Manage Employees
8 - Manage Habitats
9 - Manage Vaccines
10 - Search
0 - Sair
Escolha uma opção: Manage Habitats
1 - List all Habitats
2 - Register Habitats
3 - Change Habitat Area
4 - Change Habitat Influence over Species
5 - Plant Tree
6 - List Trees in Habitat
0 - Sair
Escolha uma opção: Habitat ID: Tree ID: Tree Name: Tree Age: Tree's Difficulty to Clean: Tree Type: (DECIDUOUS or EVERGREEN) ÁRVORE|B1|b1|2|3|DECIDUOUS|GERARFOLHAS
Manage Habitats
1 - List all Habitats
2 - Register Habitats
3 - Change Habitat Area
4 - Change Habitat Influence over Species
5 - Plant Tree
6 - List Trees in Habitat
0 - Sair
Escolha uma opção: Habitat ID: Tree ID: Tree Name: Tree Age: Tree's Difficulty to Clean: Tree Type: (DECIDUOUS or EVERGREEN) ÁRVORE|X1|x1|3|4|EVERGREEN|GERARFOLHAS
Manage Habitats
1 - List all Habitats
2 - Register Habitats
3 - Change Habitat Area
4 - Change Habitat Influence over Species
5 - Plant Tree
6 - List Trees in Habitat
0 - Sair
Escolha uma opção: Habitat ID: Tree ID: Tree Name: Tree Age: Tree's Difficulty to Clean: Tree Type: (DECIDUOUS or EVERGREEN) ÁRVORE|A1|a1|6|7|EVERGREEN|GERARFOLHAS
Manage Habitats
1 - List all Habitats
2 - Register Habitats
3 - Change Habitat Area
4 - Change Habitat Influence over Species
5 - Plant Tree
6 - List Trees in Habitat
0 - Sair
Escolha uma opção: Habitat ID: Tree ID: Tree Name: Tree Age: Tree's Difficulty to Clean: Tree Type: (DECIDUOUS or EVERGREEN) ÁRVORE|R1|r1|10|12|DECIDUOUS|GERARFOLHAS
Manage Habitats
1 - List all Habitats
2 - Register Habitats
3 - Change Habitat Area
4 - Change Habitat Influence over Species
5 - Plant Tree
6 - List Trees in Habitat
0 - Sair
Escolha uma opção: Habitat ID: ÁRVORE|A1|a1|6|7|EVERGREEN|GERARFOLHAS
ÁRVORE|B1|b1|2|3|DECIDUOUS|GERARFOLHAS
ÁRVORE|P1|Pinheiro 1|5|20|EVERGREEN|GERARFOLHAS
ÁRVORE|R1|r1|10|12|DECIDUOUS|GERARFOLHAS
ÁRVORE|X1|x1|3|4|EVERGREEN|GERARFOLHAS
Manage Habitats
1 - List all Habitats
2 - Register Habitats
3 - Change Habitat Area
4 - Change Habitat Influence over Species
5 - Plant Tree
6 - List Trees in Habitat
0 - Sair
Escolha uma opção: HABITAT|h1|nome2|12|5
ÁRVORE|A1|a1|6|7|EVERGREEN|GERARFOLHAS
ÁRVORE|B1|b1|2|3|DECIDUOUS|GERARFOLHAS
ÁRVORE|P1|Pinheiro 1|5|20|EVERGREEN|GERARFOLHAS
ÁRVORE|R1|r1|10|12|DECIDUOUS|GERARFOLHAS
ÁRVORE|X1|x1|3|4|EVERGREEN|GERARFOLHAS
Manage Habitats
1 - List all Habitats
2 - Register Habitats
3 - Change Habitat Area
4 - Change Habitat Influence over Species
5 - Plant Tree
6 - List Trees in Habitat
0 - Sair
Escolha uma opção: Main Menu
1 - New File
2 - Open File
3 - Save File
4 - Next Season
5 - Calculate Global Satisfaction
6 - Manage Animals
7 - Manage Employees
8 - Manage Habitats
9 - Manage Vaccines
10 - Search
0 - Sair
Escolha uma opção: 